README.yaml

```diff
@@ -0,0 +1,77 @@
+---
+#
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: terraform-aws-ses-lambda-forwarder
+
+# Logo for this project
+#logo: docs/logo.png
+
+# License of this project
+license: "APACHE2"
+
+# Canonical GitHub repo
+github_repo: cloudposse/terraform-aws-ses-lambda-forwarder
+
+# Badges to display
+badges:
+  - name: "Build Status"
+    image: "https://travis-ci.org/cloudposse/terraform-aws-ses-lambda-forwarder.svg?branch=master"
+    url: "https://travis-ci.org/cloudposse/terraform-aws-ses-lambda-forwarder"
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-ses-lambda-forwarder.svg"
+    url: "https://github.com/cloudposse/terraform-aws-ses-lambda-forwarder/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+# Short description of this project
+description: |-
+  Module creates email forwarder based on AWS SES and AWS lambda.
+
+# How to use this project
+usage: |-
+  ```hcl
+  variable "relay_email" {
+    default = "example@example.com"
+    description = "Email that used to relay from"
+  }
+
+  variable "forward_emails" {
+    type = "map"
+
+    default = {
+      "some_email@example.com"  = ["my_email@example.com"]
+      "other_email@example.com" = ["my_email@example.com"]
+    }
+
+    description = "Emails forward map"
+  }
+
+  module "ses" {
+    source = "git::https://github.com/cloudposse/terraform-aws-ses-lambda-forwarder.git?ref=tags/0.1.0"
+
+    namespace = "${var.namespace}"
+    name      = "${var.ses_name}"
+    stage     = "${var.stage}"
+
+    region = "${var.ses_region}"
+
+    relay_email = "${var.relay_email}"
+    domain      = "${var.parent_domain_name}"
+
+    forward_emails = "${var.forward_emails}"
+  }
+  ```
+
```

Comment:
Add a `references` section to https://www.npmjs.com/package/aws-lambda-ses-forwarder